src/components/NcAppNavigation/NcAppNavigation.vue

```diff
@@ -91,6 +93,14 @@ export default {
 			open: true,
 		}
 	},
+	computed: {
+		ariaHidden() {
+			if (this.open === true) {
+				return 'false'
+			}
+			return 'true'
```

Comment:
```suggestion
			return (!this.open).toString()
```

Reply:
I'd personally prefer `return this.open ? 'true' : 'false'`.

It is the same as `(!this.open).toString()`, but more clear to see, that we have `'true'` or `'false'` string value here.

Reply:
I don't mind, both is fine.